ten-useful-functions.js

```diff
@@ -0,0 +1,16 @@
+
+//function 1
+var total_count = 0, count = 1;
+while (count <= 10)
+{
+  total_count += count;
+  count += 1;
+}
+console.log(total_count);
+
+// function 2
+function sum(array)
+{
+    return array(function(a, b)
+    (return a + b; }, 0);
```

Comment:
This is really close, but I think you meant to use `array.reduce(function(a, b){...}, 0);`